### DISCUSSION TITLE
Is Microsoft login possible yet?

### DISCUSSION THREAD

**Discussion by BridgingDragon** [ORelio/Minecraft-Console-Client#1837] [General]
I was trying to download this on Ubuntu 20.04, Linux, and I downloaded the repository and then ran the command and it said that "msbuild" not found. Please help, thanks.

New Question: Is it possible to login with Microsoft yet?

**Comment by Ccode-lang** [comment]
You need to install Microsoft build tools.

**Comment by Ccode-lang** [comment]
Even then you will have to run the exe file you get with mono.

**Comment by Ccode-lang** [comment]
I recommend that you get a bleeding edge build and just install mono.

**Reply by BridgingDragon** [reply]
could you explain how i do that

**Reply by Ccode-lang** [reply]
Go to releases on GitHub and download a .exe.

**Reply by Ccode-lang** [reply]
For how to download mono it is different for all systems and I have Debian so I don't know. Try searching it.

**Reply by BridgingDragon** [reply]
Edit : Figured it out thanks

**Reply by Ccode-lang** [reply]
Your welcome. 😊
